File: aeongseo/SWEA/CPP/swea2072.cpp
#include <iostream>
using namespace std;

int main(){
    int test_case;
    int T;

    cin>>T;

    for(test_case=1;test_case<=T; ++test_case){
        int arr[10];
        int sum = 0;
    
        for(int i =0; i<10; i++){
            cin>>arr[i];
        }
    
        for(int j=0; j<10; j++){
            if(arr[j] % 2 != 0){                        // 값이 홀수면
                sum += arr[j];                          // sum에 더하기
            }
        }
    
        cout<<"#" << test_case << " " << sum << "\n";

    }
    return 0;

}

File: aeongseo/SWEA/PYTHON/swea2072.py
'''
2072. 홀수만 더하기
'''

T = int(input())

for test_case in range(1, T+1):
    arr = list(map(int, input().split()))   # 값 입력
    sum = 0 

    for i in range(10):                     # 입력받은 정수 10개의 값에서
        if arr[i] % 2 != 0:                 # 값이 홀수라면
            sum += arr[i]                   # sum에 더하기

    print(f'#{test_case} {sum}')